12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1,8 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+"""
+excersize #1
+Print list of all English lowercase letters
+"""
+
+x = range(ord('a'),ord('z')+1)
+for i in x: print chr(i)
+

```

Comment:
Please use list comprehension to generate the list:
https://www.tocode.co.il/bundles/python-verint/lessons/list-comprehension